rocketchat-common/src/main/java/com/rocketchat/common/data/lightstream/document/RocketChatRolesDocument.java

```diff
@@ -0,0 +1,88 @@
+package com.rocketchat.common.data.lightstream.document;
+
+import java.util.Date;
+import org.json.JSONObject;
+
+public class RocketChatRolesDocument {
+    String name;
+    String scope;
+    String description;
+    Boolean _protected;
+    Date updatedAt;
+
+    public RocketChatRolesDocument(JSONObject roles) {
+        name = roles.optString("name");
+        scope = roles.optString("scope");
+        description = roles.optString("description");
+        _protected = roles.optBoolean("protected");
+        if (roles.opt("_updatedAt") != null) {
+            updatedAt = new Date(roles.optJSONObject("_updatedAt").optLong("$date"));
+        }
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public String getScope() {
+        return scope;
+    }
+
+    public void setScope(String scope) {
+        this.scope = scope;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public Boolean get_protected() {
```

Comment:
We don't need to follow exactly the JSON wire format, `get_protected()` is not using the same naming pattern as `getDescription()`